### PR TITLE
feat: policies list in the quick start page.

### LINKED ISSUE #401: Add ArtifactHub packages to website
As part of our ongoing efforts in encouraging better developer experience across different CNCF projects, we figured it would be nice for projects to [showcase their packages hosted on ArtifactHub by embedding directly on the website](https://github.com/cncf/techdocs/issues/213).

To embed all the necessary project-related packages, kindly follow [this guide](https://artifacthub.io/docs/topics/embedding_artifacts/). You can as well view some examples:
- [Keptn integration](https://github.com/keptn/keptn.github.io/pull/929)
- [Keda](https://github.com/kedacore/keda-docs/pull/805/commits/b0d34a82cd8cd3f2a6833e98f88a13c8622b21e8)

### PR DESCRIPTION
**Pull request by jvanz**
## Description

Adds an embedded HTML with the policies available in the ArtifactHub

Fix #401

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -408,4 +409,12 @@ kubectl delete -l "kubewarden" mutatingwebhookconfigurations.admissionregistrati
 
 ## What's next?
 
-Now, you are ready to deploy Kubewarden! Have a look at the policies on [artifacthub.io](https://artifacthub.io/packages/search?kind=13), on [GitHub](https://github.com/topics/kubewarden-policy), or reuse existing Rego policies as shown in the [following chapters](tutorials/writing-policies/rego/01-intro-rego.md).
+Now, you are ready to deploy Kubewarden! Have a look at the policies on
+[artifacthub.io](https://artifacthub.io/packages/search?kind=13), on
+[GitHub](https://github.com/topics/kubewarden-policy), or reuse existing Rego
+policies as shown in the [following
+chapters](tutorials/writing-policies/rego/01-intro-rego.md). Here is the list of
+available policies on ArtifactHub:
```
Comment (flavio) [inline docs/quick-start.md:417]:
```suggestion
chapters](tutorials/writing-policies/rego/01-intro-rego.md).

Finally, here is the list of available policies on ArtifactHub.
```

**Review by flavio — COMMENTED** [review]
Looks nice, I left a minor fix

**Review by viccuad — APPROVED** [review]
LGTM modulo Flavio's suggestion.

File: docs/quick-start.md [hunk 2]
```diff
@@ -408,4 +409,12 @@ kubectl delete -l "kubewarden" mutatingwebhookconfigurations.admissionregistrati
...
+
+<div class="artifacthub-widget-group" data-url="https://artifacthub.io/packages/search?kind=13&sort=relevance&page=1" data-theme="light" data-header="false" data-stars="false" data-color="#fe7c3f" data-responsive="true"  data-loading="true"></div>
+
```
Comment (jhkrug) [inline docs/quick-start.md:420]:
```suggestion
Now, you are ready to deploy Kubewarden! Have a look at the policies on
[artifacthub.io](https://artifacthub.io/packages/search?kind=13), on
[GitHub](https://github.com/topics/kubewarden-policy), or reuse existing Rego
policies as shown in the [following
chapters](tutorials/writing-policies/rego/01-intro-rego.md). 

<details>
<summary>Full list of available policies on ArtifactHub</summary>

<div class="artifacthub-widget-group" data-url="https://artifacthub.io/packages/search?kind=13&sort=relevance&page=1" data-theme="light" data-header="false" data-stars="false" data-color="#fe7c3f" data-responsive="true"  data-loading="true"></div>

</details>

```

Hi @jvanz, it's a long list. Are you open to having it in a details wrapper?

File: docs/quick-start.md (hunk 1 above)
Comment (jvanz) [reply docs/quick-start.md]: I think the @jhkrug approach a better way to list the policies and makes this suggestion obsolete. Therefore, I'll skip this change. Ok? ;)

**Review by jhkrug — APPROVED** [review]
lgtm

**Comment by flavio** [comment]
I prefer @jhkrug's approach. You can go ahead and merge this PR

**Review by fabriziosestito — APPROVED** [review]

**Comment by thisisobate** [comment]
@jvanz I think it would be cool to embed as a widget rather than linking to artifactHub. The idea is to make it easier for users to directly interact with Kubewarden policies directly on kubewarden.io. 

You can take a look at some examples here: 
- [Keptn integration](https://github.com/keptn/keptn.github.io/pull/929)
- [Keda](https://github.com/kedacore/keda-docs/pull/805/commits/b0d34a82cd8cd3f2a6833e98f88a13c8622b21e8)

**Pull request merged**